google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/Util.java

```diff
@@ -197,4 +199,18 @@ static void recordMetricsFromMetadata(
     // Record gfe metrics
     tracer.recordGfeMetadata(latency, throwable);
   }
+
+  static GrpcCallContext addHandlerToCallContext(
```

Comment:
Method name is too generic. You can only know what handler is when you read the method body. The method name should should directed to the caller as the audience. Maybe injectBigtableStreamTracer?

Reply:
Also ad javadoc that explains that this bridges gRPC's stream tracing to bigtable tracing